### DISCUSSION TITLE
News archive by year and month group

### DISCUSSION THREAD

**Discussion by shobhithn7** [statamic/cms#5456] [Q&A]
I want to display archive for my news page
 ![Screenshot_8](https://user-images.githubusercontent.com/77607972/157694332-c0101e62-518f-4eb0-bd9b-92528b69d0c5.png)
, like this. can any one please help me. below is my code.

```
{{ collection:news as="entries" }}
    {{ entries group_by="date|F Y" }}
        {{ groups }}
            <h1>{{ group }}</h1>
            <ul>
                {{ items }}
                    <li>{{ title }}</li>
                {{ /items }}
            </ul>
        {{ /groups }}
    {{ /entries }}
{{ /collection:news }}
```

**Comment by edalzell** [comment]
What's not working?

**Reply by shobhithn7** [reply]
wirking fine i want result as above image, I don't know how to loop the group with another group.
